pillar.example

```diff
@@ -12,6 +12,7 @@ rsyslog:
   listenudp: true                         # omit to disable listening on udp port
   imkllog: true                           # omit to log kernel messages
   imjournal: false                        # omit to log journal messages
+  markmessageperiod: 1200                 # Specifies when mark messages are written (seconds)
```

Comment:
Mind you add an `immark: false` entry here, so it's clear that it needs to be set to `true` to enable it?